### PR TITLE
Make the "online-update-configuration" module compile

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

File: patches/online-update-configuration.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff --git a/src/clients/online_update_configuration.ycp b/src/clients/online_update_configuration.ycp
+index 997ee11..62b8590 100644
+--- a/src/clients/online_update_configuration.ycp
++++ b/src/clients/online_update_configuration.ycp
+@@ -44,7 +44,7 @@ you may find current contact information at www.novell.com
+     import "CommandLine";
+     import "Installation";
+ 
+-    include "online-update-configuration/OUCDialogs.ycp";
++    include "../include/OUCDialogs.ycp";
```
Comment (jreidinger) [inline patches/online-update-configuration.patch:10]: thats wrong to use relative path. Please make it 

```
include "OUCDialogs.ycp";
```

File: patches/online-update-configuration.patch [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+diff --git a/src/clients/online_update_configuration_auto.ycp b/src/clients/online_update_configuration_auto.ycp
+index 102589e..740f7f0 100644
+--- a/src/clients/online_update_configuration_auto.ycp
++++ b/src/clients/online_update_configuration_auto.ycp
+@@ -47,7 +47,7 @@ you may find current contact information at www.novell.com
+     import "PackageCallbacks";
+ */
+ 
+-    include "online-update-configuration/OUCDialogs.ycp";
++    include "../include/OUCDialogs.ycp";
```
Comment (jreidinger) [inline patches/online-update-configuration.patch:23]: same here...
thats lead to probably mistake made by you.
OUCDialogs.ycp should go to src/include/online-update-configuration/
so you preserve paths as it was.

File: data/online-update-configuration.yml [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+  - yast2
 
-files:
-  - src/clients/online_update_configuration.ycp
-  - src/clients/online_update_configuration_auto.ycp
-  - src/dialogs/OUCDialogs.ycp
-  - src/modules/OnlineUpdateConfiguration.ycp
+moves:
+  - from: "src/dialogs/OUCDialogs.ycp"
+    to:   src/include
```
Comment (jreidinger) [inline data/online-update-configuration.yml:6]: wrong, see below

**Comment by dmajda** [comment]
I did changes requested by @jreidinger in #36, closing this request.

**Pull request closed without merging**